proposals/stage-1/TRAVEL_FUND/travel-visas.md

```diff
@@ -70,7 +70,7 @@ plans.
 
 ## Need some assistance?
 
-We are adding all VISA related information in this document but If you need more assistance or you have any question for your application, feel free to email us at [visainfo@openjsf.org ](mailto:visainfo@openjsf.org). We are not VISA experts but will try
+We are adding all VISA related information in this document but If you need more assistance or you have any question for your application, feel free to email us at [visainfo@openjsf.org](mailto:visainfo@openjsf.org). We are not VISA experts but will try
 to assist as much as we can.
```

Comment:
I'm not thoroughly familiar with `remark-lint` rules, but I'm a little surprised that e.g. this line isn't getting merged with the preceding. Or is the intent to first add the linter with minimal rules, and then separately argue about what rules should be enabled?

Reply:
> is the intent to first add the linter with minimal rules, and then separately argue about what rules should be enabled?

I would prefer this.